L/libCEED/build_tarballs.jl

```diff
@@ -0,0 +1,34 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libCEED"
+version = v"0.7.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/CEED/libCEED.git", "6134893e0067dc2a87bbbe22a551ae134ad5b885")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libCEED
+make -j${nproc} MEMCHK=0 CC_VENDOR=gcc
+make install
```

Comment:
Same argument also to `make install`:
```suggestion
make -j${nproc} MEMCHK=0 CC_VENDOR=cc
make install MEMCHK=0 CC_VENDOR=cc
```
Also, don't force GCC as compiler, when building for FreeBSD and macOS we don't want to use that, if possible

Reply:
The variable `CC_VENDOR` determines the compiler flags, but not the compiler itself. `CC_VENDOR=cc` will not have an effect. The flags for gcc and clang are the same, so I think this should be fine. There is some logic in the Makefile to set `CC_VENDOR` automatically based on the output of `$(CC) --version`, but it was not working properly.

Reply:
Ok, looking at the log it's actually building with `cc`, which is what we want.  All good then, thanks!